### ISSUE TITLE
before:function load() and after:function load() fire twice using scrollify.move(id);

### ISSUE THREAD

**Issue by vanderLo** [lukehaas/Scrollify#65]
I'm using click events with both e.preventDefault(); and e.stopImmediatePropagation(); or return false; to prevent multiple firing. 

But I can still observe multiple events, defined both within the before and after function (simple alerts), firing from this. How come?

````	$("#asd").click(function(e) {
````	  e.preventDefault();
````	  e.stopImmediatePropagation();
````	$.scrollify.move('#asd');
````	});

...in correct context:
````before:function load() { alert('first'); },
````after:function load() { alert('second'); }

When using regular scrolling, then both events fire only once. Using the click function, I get 4 alerts: 2x first and 2x second. Same thing happens btw after resizing the window.

**Comment by lukehaas** [comment]
Thanks for letting me know about this. I'll include a fix in the next update.

**Comment by lukehaas** [comment]
This should be fixed now.

**Comment by vanderLo** [comment]
Thanks! +1 
Works fine now!

**Comment by vanderLo** [comment]
The first issue is solved, but unfortunately resizing the window does still fire the alerts.

**Comment by lukehaas** [comment]
This should now be resolved in version 0.1.13
